route.go

```diff
@@ -396,7 +396,7 @@ func (r *Route) Schemes(schemes ...string) *Route {
 	for k, v := range schemes {
 		schemes[k] = strings.ToLower(v)
 	}
-	if r.buildScheme == "" && len(schemes) > 0 {
+	if r.getBuildScheme() == "" && len(schemes) > 0 {
```

Comment:
I'm not sure if this is correct. When `Schemes` is called it should set the schemes for the route, regardless of what the parent's schemes are. In this case the build scheme would always be the parent's, even if the route's schemes didn't include it.